python/ray/tests/unit/BUILD

```diff
@@ -0,0 +1,8 @@
+py_test(
+    name = "runtime_env_validation_unit_test",
```

Comment:
I think the naming convention should be test_runtime_env_validation.py?

Reply:
SG, updated.